share/picongpu/examples/Bremsstrahlung/include/picongpu/param/particle.param

```diff
@@ -22,8 +22,7 @@
 #include "picongpu/particles/startPosition/functors.def"
 #include "picongpu/particles/manipulators/manipulators.def"
 
-#include <pmacc/nvidia/functors/Add.hpp>
-#include <pmacc/nvidia/functors/Assign.hpp>
+#include <pmacc/math/operation.hpp>
```

Comment:
I think this is unused and can be removed.

Reply:
I would keep it in to avoid compile issues if someone is cloning from this example and copy-pasting functors from other examples.